docs/dev_guide/contents/tests.rst

```diff
@@ -23,7 +23,7 @@ Since the testing dependencies are not actually required to install or use sunpy
 
 Developers who want to run the test suite will need to install the testing packages using pip::
 
-    $ pip install -e .[tests]
+    $ pip install -e ".[tests]""
```

Comment:
Double quote is a typo?

Reply:
How did I miss that?!

Reply:
Oops!

Reply:
Stupid double quote completion.

Reply:
If you only accepted my suggestion

Reply:
If you have time, you can throw up another PR otherwise I'll get to it when I end up at UIC.